ctk/common/src/main/kotlin/org/codice/compliance/verification/profile/ProfilesVerifier.kt

```diff
@@ -50,61 +40,4 @@ class ProfilesVerifier(private val node: Node) {
                     node = node)
         }
     }
-
-    /**
-     * Verify response against the Profiles Spec document
-     */
-    fun verify() {
-        verifyHolderOfKey()
-    }
-
-    /**
-     * 3.1 Holder of Key
```

Comment:
Moved Holder of Key Verification to `SingleSignOnProfileVerifier` because it's specific to SSO Responses since Logout Responses cannot contain subjects